actors/user/src/main/java/org/sunbird/user/actors/EducationManagementActor.java

```diff
@@ -0,0 +1,233 @@
+
+@ActorConfig(
+  tasks = {},
+  asyncTasks = {"upsertUserEducation"}
+)
+public class EducationManagementActor extends BaseActor {
+
+  private CassandraOperation cassandraOperation = ServiceFactory.getInstance();
+  private Util.DbInfo addrDbInfo = Util.dbInfoMap.get(JsonKey.ADDRESS_DB);
+  private Util.DbInfo eduDbInfo = Util.dbInfoMap.get(JsonKey.EDUCATION_DB);
+
+  @Override
+  public void onReceive(Request request) throws Throwable {
+    if (UserActorOperations.UPSERT_USER_EDUCATION
+        .getValue()
+        .equalsIgnoreCase(request.getOperation())) {
+      upsertEducationDetails(request);
+    } else {
+      ProjectCommonException exception =
+          new ProjectCommonException(
+              ResponseCode.invalidOperationName.getErrorCode(),
+              ResponseCode.invalidOperationName.getErrorMessage(),
+              ResponseCode.CLIENT_ERROR.getResponseCode());
+      sender().tell(exception, self());
+    }
+  }
+
+  @SuppressWarnings("unchecked")
+  private void upsertEducationDetails(Request request) {
+    Map<String, Object> requestMap = request.getRequest();
+    String operationtype = (String) requestMap.get(JsonKey.OPERATION_TYPE);
+    List<Map<String, Object>> reqList =
+        (List<Map<String, Object>>) requestMap.get(JsonKey.EDUCATION);
+
+    for (int i = 0; i < reqList.size(); i++) {
+      Map<String, Object> educationDetailsMap = reqList.get(i);
+      String createdBy = (String) requestMap.get(JsonKey.ID);
+      Response addrResponse = null;
+      if (JsonKey.CREATE.equalsIgnoreCase(operationtype)) {
+        educationDetailsMap.put(JsonKey.ID, ProjectUtil.getUniqueIdFromTimestamp(i));
+        if (educationDetailsMap.containsKey(JsonKey.ADDRESS)) {
+          addrResponse = upsertEducationAddressDetails(educationDetailsMap, createdBy);
+        }
+        insertEducationDetails(requestMap, educationDetailsMap, addrResponse, createdBy);
+      } else {
+        if (educationDetailsMap.containsKey(JsonKey.IS_DELETED)
+            && null != educationDetailsMap.get(JsonKey.IS_DELETED)
+            && ((boolean) educationDetailsMap.get(JsonKey.IS_DELETED))
+            && !StringUtils.isBlank((String) educationDetailsMap.get(JsonKey.ID))) {
+          deleteEducationDetails(educationDetailsMap);
+          continue;
+        }
+        if (educationDetailsMap.containsKey(JsonKey.ADDRESS)) {
+          addrResponse = upsertEducationAddressDetails(educationDetailsMap, createdBy);
+        }
+        updateEducationDetails(educationDetailsMap, addrResponse, createdBy);
+      }
+    }
+    saveUserEducationToEs(requestMap);
+  }
+
+  private void saveUserEducationToEs(Map<String, Object> requestMap) {
+    Request userRequest = new Request();
+    userRequest.setOperation(UserActorOperations.UPSERT_USER_EDUCATION_TO_ES.getValue());
+    userRequest.getRequest().putAll(requestMap);
+    tellToAnother(userRequest);
+  }
+
+  private void updateEducationDetails(
```

Comment:
Create EducationDao and EducationDaoImpl with below methods.

insert
update
upsert
delete